src/pages/management/changelog/index.mdx

```diff
@@ -25,9 +25,14 @@ The developer preview version provides a preview of the upcoming changes to the
 
 ## [v3.5] - Developer preview
 
+## General
+
+- There's a new type of request, [batch requests](/management/configuration-api/v3.5/#batch-requests). They allow for sending multiple calls of the same method in a single request.
+
 ### Agents
 
 - The [**List Agents**](/management/configuration-api/v3.5/#list-agents) method can now filter agents by their `suspended` status.
+- The following methods now support batch requests: [Create Agent](#create-agent), [Update Agent](#update-agent), [Delete Agent](#delete-agent), [Suspend Agent](#suspend-agent), [Unsuspend Agent](#unsuspend-agent), and [Approve Agent](#approve-agent).
```

Comment:
Hello @litwinow :) Ola asked me to publish this PR as she approved them. However, just to make sure, I checked the preview and saw that the links to those methods do not work. Ola is on holiday, so I just wanted to confirm with you if I can publish it (maybe there was some agreement between you two that I am not aware of) :p

Reply:
Good catch @kamilatokarska , I'll fix the links today. Also let's wait with the publishing a few days as the code that implements the batches is not yet on production.